book/07-git-tools/sections/interactive-staging.asc

```diff
@@ -21,14 +21,14 @@ $ git add -i
 What now>
 ----
 
-You can see that this command shows you a much different view of your staging area than you're probably used to -- basically, the same information you get with `git status` but a bit more succinct and informative.
-It lists the changes you’ve staged on the left and unstaged changes on the right.
+Makikita mo na ang utos na ito ay nagpapakita sayo ng isang mas magkaiba na tingnan sa iyong staging na lugar kayasa sa marahil na ginagamit sa -- talagang, ang parehong impormasyon na iyong makukuha gamit ang `git status` na ngunit medyo maikli at nakakapagturo.
+Ito ay naglilista ng mga pagbabago na iyong nai-stage sa kaliwa at mga naka-unstage na mga pagbabago sa iyong kanan.
 
-After this comes a ``Commands'' section, which allows you to do a number of things like staging and unstaging files, staging parts of files, adding untracked files, and displaying diffs of what has been staged.
+Pagkatapos ito ay mayroon ding isang ``Commands'' na seksyon, na kung saan ay nagpapahintulot sayo na gawin ang bilang ng mga bagay katulad ng staging at unstaging na mga file, ang staging na mga parte ng mga file, ang pagdagdag ng untracked na mga file, at ang pagpakita ng mga diff displaying diffs sa kung ano ang na-staged.
 
-==== Staging and Unstaging Files
+==== Ang Pag-staging at Unstaging na mga File
 
-If you type `2` or `u` at the `What now>` prompt, you're prompted for which files you want to stage:
+Kung ikaw ay nag-tpe ng `2` o `u` sa `What now>` prompt, ikaw ay sinesenyasan kung anong mga file na gusto mong i-stage:
 
```

Comment:
"ikaw ay nag-tpe" change it to "ikaw ang nag-type" typo on type.

Reply:
done change